website/content/docs/agent/caching/persistent-caches/kubernetes.mdx

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+page_title: Vault Agent Kubernetes Persistent Cache
```

Comment:
Nit: Kubernetes - Vault Agent Persistent Cache

(Similar format to auth method titles)

Reply:
Good call, fixed.